### ISSUE TITLE
mc.exe produces weird output during Jenkins jobs

### ISSUE THREAD

**Issue by harshavardhana** [minio/mc#2028]
Jenkins jobs provide a "console log / output" from the steps it runs during a build. I'm using mc.exe to copy some build artifacts into a workspace before doing a build.

## Expected Behavior
```
"myminio/temp/xmlpdf11.dll" -> "c:/jenkins/workspace/SomeBuild/temp/xmlpdf11.dll"
```

## Current Behavior

```
â€˜myminio/temp/xmlpdf11.dllâ€™ -> â€˜c:/jenkins/workspace/SomeBuild/temp/xmlpdf11.dllâ€™
```

## Possible Solution

Detect terminal or lack thereof and produce legible/clear output for this scenario.

## Steps to Reproduce (for bugs)

1. Create a Jenkins master on Windows (maybe just run the Jenkinsci Docker image)
2. Create a job with a Windows Batch build step and invoke `mc.exe cp`

## Context

This provides confusing output for people using the builds and additional non-sense characters in the build output.

## Your Environment

- Windows 2012 R2 / 6.3
- Terminal: cmd.exe
- mc version:

```
Version: 2017-02-06T20:16:19Z
Release-tag: RELEASE.2017-02-06T20-16-19Z
Commit-id: 2c8115de4edc5612525488b1b3b804689d336d01
```

**Comment by vadmeste** [comment]
@harshavardhana, can't we just use simple characters like ` ?

**Comment by harshavardhana** [comment]
> @harshavardhana, can't we just use simple characters like ` ?

That is better to avoid the problems of dumb terminals.

**Comment by harshavardhana** [comment]
Fixed by #2033
